Managers/PrimitiveMan.h

```diff
@@ -138,6 +138,44 @@ namespace RTE {
 		/// <param name="color">Color to draw primitive with.</param>
 		void DrawBoxFillPrimitive(short player, Vector start, Vector end, unsigned char color) { m_Primitives.push_back(new BoxFillPrimitive(player, start, end, color)); }
 
+		/// <summary>
+		/// Schedule to draw a round box primitive.
+		/// </summary>
+		/// <param name="start">Start position of primitive in scene coordinates.</param>
```

Comment:
Start is top left and end is bottom right, right? Might be nice to rename some params for this and similar primitives to make this clearer.